packages/dx-react-core/src/forward-ref.jsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+export class ForwardRef extends React.PureComponent {
```

Comment:
This component seems weird. Maybe we can rename it to something more definitive.